packages/manager/modules/pci/src/projects/creating/controller.js

```diff
@@ -31,7 +31,31 @@ export default class PciProjectCreatingCtrl {
   getDeliveredProjectId() {
     return this.pciProjectCreating
       .getOrderDetails(this.orderId)
-      .then((details) => get(details, '[0].domain'));
+      .then((details) => {
+        const orderItemsDetailPromises = details.map(({ orderDetailId }) =>
+          this.pciProjectCreating
+            .getOrderItemDetails(this.orderId, orderDetailId)
+            .then((data) => {
+              return { orderDetailId, item: data };
+            }),
```

Comment:
Looks like you map ids to fetch details, can't we use Iceberg instead for this ?

Reply:
are there any particular reasons to use iceberg ?
I search for iceberg usage, but i not see something related to map and iceberg.
May be you had a sample usage ?

Reply:
Yes to replace N+1 calls by 1 
You can use https://github.com/ovh-ux/ng-ovh-api-wrappers

Reply:
Hello @marie-j , i take look about iceberg but in my case i am note sure i can implement it.
For me iceberg can be used in those cases: filter, sort and aggregate.

Tell me if you have an idea.

Reply:
Aren't we aggregating here ? For each ids we fetch some details

Reply:
I am not sure i can use iceberg to make 1 call instead of 4 calls.
If i summaries, we have:
1 - call to .getOrderDetails(this.orderId) to get details for each ordered item
2 - i get the ids for each of those ordered id
3 - For each orderDetailId i make a call to /.../extension
4 - I got the right item

So, @marie-j  where and how can i use iceberg ?